### DISCUSSION TITLE
Rename multiple files

### DISCUSSION THREAD

**Discussion by ag0nyr0se** [aleksey-hoffman/sigma-file-manager#246] [Ideas]
There's the option to rename individual files, but can there be a bulk rename function added? Not a full on regular expression renamer, but maybe search all files containing "xyz" and rename them to sequential (xyz1 zyx,2, xyz3, etc) or something lightweight. 

Just an idea for convenient file and data management.

**Comment by aleksey-hoffman** [comment]
I'm planning to add the bulk rename feature somewhere in v1.8 - v1.10 update. It will support glob pattern searching / regex.

Thanks for the suggestion! 
I should probably create the list of all planned features here on Github
